### PR TITLE
Fix footerhk styles

### PR DESCRIPTION
**Pull request by swetharajendra05**
** Please provide as much detail as possible, but feel free to remove irrelevant sections **

{{ Reason for change }}
- fix the listitem to padding/line-height as per seekasia standard spec from design team
- updated the footer styles

BREAKING CHANGE:

{{ Description of breaking API change }}

RFC URL:

{{ URL of associated RFC for API change }}

MIGRATION GUIDE:

Before:

```js
{{ Old code }}
```

After:

```js
{{ New code }}
```

EXAMPLE USAGE:

```js
{{ Code }}
```

### CODE REVIEW

File: react/ListItem/ListItem.less [hunk 1]
```diff
@@ -1,22 +1,28 @@
 @import (reference) "~seek-asia-style-guide/theme";
-@list-item-h-padding: @grid-base * 4;
-@list-item-v-padding: @grid-base * 3;
+
+@list-item-padding: @grid-base * 3;
+@list-item-padding-compact: @grid-base * 2;
+@list-item-line-height-compact: @grid-base * 4.5; //this is as per brandportal spec
```
Comment (ChanJit) [inline react/ListItem/ListItem.less]: `@grid-base * 4 + 2`?

Comment (swetharajendra05) [reply react/ListItem/ListItem.less]: yeah, we can but not sure which is the right approach:)

File: react/ListItem/ListItem.less [hunk 2]
```diff
@@ -1,22 +1,28 @@
 @import (reference) "~seek-asia-style-guide/theme";
-@list-item-h-padding: @grid-base * 4;
-@list-item-v-padding: @grid-base * 3;
+
+@list-item-padding: @grid-base * 3;
```
Comment (ChanJit) [inline react/ListItem/ListItem.less]: may be just keep this one and remove  `@list-item-icon-padding-right` cause kind of similar

File: react/ListItem/ListItem.less (hunk 1 above)
Comment (ChanJit) [reply react/ListItem/ListItem.less]: well currently @grid-base is 4 what if it turn to 5? using `+` is just to ensure it will return a whole number.

**Pull request merged**